rpms/awips2.edex/Installer.edex/component.spec

```diff
@@ -62,24 +62,10 @@ if [ $? -ne 0 ]; then
    exit 1
 fi
 
```

Comment:
We don't need to create the ndm directory here anymore?

Reply:
No, I don't think those entries did anything. The ndm directory got created below when the files were copied in (below).